src/custom-select/index.ts

```diff
@@ -17,41 +17,39 @@ class CustomSelect {
         this.$select.onSelect(event.item.i);
       }
       this.update({ isActive: false });
-      this.refs.toggle.blur();
     }
   };
   isActive: boolean = false;
 
+  init() {
+    utils.WINDOW.addEventListener('click', this.onClickDeactivate);
```

Comment:
I'd like to have it so that the event listener only exists on `window` when there's actually the potential for the dropdown to be closed. Naïvely this means adding a `removeEventListener()` method to `WINDOW`, but instead I'd like you to refactor `WINDOW` in `@storefront/core` such that it is a function itself that returns `window`. That way we'll never have to modify it in the future.

The only other component using `WINDOW` is `@storefront/sayt`, so it will need to be updated as well